lib/trailblazer/operation/public_call.rb

```diff
@@ -46,13 +61,13 @@ def options_for_public_call(*args)
 
     # Compile a Context object to be passed into the Activity::call.
     # @private
-    def self.options_for_public_call(options, **flow_options)
+    def self.options_for_public_call(options, flow_options = {})
       Trailblazer::Context(options, {}, flow_options[:context_options])
     end
 
     # @semi=public
-    def flow_options
-      {}
+    def flow_options(options = {})
```

Comment:
Shouldn't that be named `original_flow_options` to make it clear what it is? Or maybe I'm wrong...

Reply:
Good idea, would it make mode clear actually.
I've update it. Just need to update it in your 🚓 project I guess.